test/e2e/connection.spec.ts

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ElectronApplication, Page, test, expect } from '@playwright/test';
+import { launchVsCodeWithMssqlExtension } from './utils/utils';
+
+test.describe('MSSQL Extension - Database Connection', async () => {
+	let vsCodeApp: ElectronApplication;
+	let vsCodePage: Page;
+
+	test.beforeAll(async () => {
+		const { electronApp, page } = await launchVsCodeWithMssqlExtension();
+		vsCodeApp = electronApp;
+		vsCodePage = page;
+	});
+
+	test('Connect to local SQL Database, and disconnect', async () => {
+		// wait for 30 seconds
+		const addConnectionButton = await vsCodePage.locator('div[aria-label="Add Connection"]');
+		let isConnectionButtonVisible = await addConnectionButton.isVisible();
+		if (!isConnectionButtonVisible) {
+			await vsCodePage.click('a[aria-label="SQL Server (Ctrl+Alt+D)"]');
+		}
+
+		await expect(addConnectionButton).toBeVisible({ timeout: 10000 });
+		await addConnectionButton.click();
+
+		// await vsCodePage.click('input[aria-label="input"]');
+		await vsCodePage.fill('input[aria-label="input"]', '(localdb)\\MSSqlLocalDb');
```

Comment:
Ideally, we would like to take the connection details from command line and fill them up here instead of hard coding it in the test. I am guessing when we add this to pipelines, it will take the server url and password for the local docker db that we spin up for that run. It will be nice to make a class that does this so that we can reuse it across all tests.